### PR TITLE
fix(curriculum): some typos

### PR DESCRIPTION
**Pull request by yeger00**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-tribute-page/bd7158d8c442eddfaeb5bd18.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ assert(leftMargin - rightMargin < 6 && rightMargin - leftMargin < 6)
           </li>
           <li>
             <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat
-            with a high-yeidling American breed, creating a strain that responds
+            with a high-yeilding American breed, creating a strain that responds
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lab-tribute-page/bd7158d8c442eddfaeb5bd18.md:281]:
```suggestion
            with a high-yielding American breed, creating a strain that responds
```

File: tools/scripts/seed-exams/README.md
Comment (gikf) [inline tools/scripts/seed-exams/README.md:1]: Line 31 also has similar typo.

**Comment by yeger00** [comment]
Thanks for your review, I missed those

**Review by Sembauke — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Pull request merged**